azure-functions/acbs-function/mappings/facility/helpers/get-exposure-period.test.js

```diff
@@ -0,0 +1,237 @@
+const { cloneDeep } = require('lodash');
+const getExposurePeriod = require('./get-exposure-period');
+const { PRODUCT } = require('../../../constants');
+
+describe('getExposurePeriod', () => {
+  describe('with amendment and fmr', () => {
+    const dealType = PRODUCT.TYPE.BSS_EWCS;
+    /**
+     * mock data taken from ACBSIssueFacilityResponseBody
+     * in external-api\src\utils\swagger-definitions\acbs.ts
+     */
+    const mockFmr = {
+      // submittedToACBS: '2021-09-30T09:42:31+01:00',
```

Comment:
Undecided on whether YAGNI and get rid of this, or add it to a mocks file to save searching through swagger docs again